test/e2e/suite_test.go

```diff
@@ -41,11 +42,8 @@ import (
 // These tests use Ginkgo (BDD-style Go testing framework). Refer to
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
-var cfg *rest.Config
 var k8sClient client.Client
-var testEnv *envtest.Environment
 var ctx context.Context
```

Comment:
Can we get rid of this package level ctx and use ctx which is passed in suite by ginkgo?

Reply:
Yep, will do later.